public/dist/data/helper-scripts/partners.js

```diff
@@ -0,0 +1,25 @@
+var $partners = $('.partners-list').find('li');
+var partnerList = [];
+
+function getPartnerData(index) {
```

Comment:
Global function - please namespace this.

Reply:
done